articles/faq/recipient-address-was-suppressed-due-to-customer-policy.md

```diff
@@ -24,8 +24,10 @@ You can use the API key to view who was on your account's Suppression List, but
 
 You can edit that by visiting this URL and clicking the Edit button for that particular API key and adding the Suppression Lists Read/Write permission: [https://app.sparkpost.com/#/account/credentials](https://app.sparkpost.com/#/account/credentials)
 
+Please see this artcile on how to manage your suppression list:
+https://www.sparkpost.com/docs/user-guide/using-suppression-lists/
+
 Some workarounds for this situation are:
 
-1. Try sending to a different single email address (perhaps your own).
-2. Find new ways to re-engage that customer on other channels such as your website, SMS (with their permission), social channels, etc...
-3. If this is a known testing address of your own, you can remove the email address from your account's Suppression List, but ONLY do this if you KNOW that this email address is yours and used ONLY for development. Otherwise you will seriously damage your reputation with the customer and Sparkpost.com.
+1. Find new ways to re-engage that customer on other channels such as your website, SMS (with their permission), social channels, etc...
+2. If this is a known testing address of your own, you can remove the email address from your account's Suppression List, but ONLY do this if you KNOW that this email address is yours and used ONLY for development. Otherwise you will seriously damage your reputation with the customer and SparkPost
```

Comment:
Do we need the all-caps words? It feels a bit shouty to me.

Full stop `.` after `...and SparkPost` ?